anda/tools/portage/anda.hcl

```
@@ -0,0 +1,5 @@
+project pkg {
```

Since this package is `noarch`, you only need to build it once:

```suggestion
project pkg {
    arches = ["x86_64"]
```

---

It's not noarch. This is a binary package.